tests/ui/test_ui.py

```diff
@@ -230,56 +230,60 @@ def just_set_message_view(self: Any) -> None:
         show_left_panel.assert_called_once_with(visible=True)
 
     @pytest.mark.parametrize("autohide", [True, False])
-    @pytest.mark.parametrize("visible, width", [(True, LEFT_WIDTH), (False, TAB_WIDTH)])
-    def test_show_left_panel(
+    @pytest.mark.parametrize("visible", [True, False])
+    @pytest.mark.parametrize("test_method", ["left_panel", "right_panel"])
```

Comment:
While there may be scope for combining tests, this is best to do as a separate refactor.

Regarding the reuse I was referring to in another comment which may have resulted in this change, I meant you have a common end of list that you can use to assert against for the left and right cases. So you can use `left_specific_part + common_part` in the comparison.